chart/flux/README.md

```diff
@@ -185,43 +185,51 @@ The following tables lists the configurable parameters of the Weave Flux chart a
 | `git.setAuthor` | If set, the author of git commits will reflect the user who initiated the commit and will differ from the git committer. | `false`
 | `git.label` | Label to keep track of sync progress, used to tag the Git branch | `flux-sync`
 | `git.ciSkip` | Append "[ci skip]" to commit messages so that CI will skip builds | `false`
 | `git.pollInterval` | Period at which to poll git repo for new commits | `5m`
 | `git.timeout` | Duration after which git operations time out | `20s`
-| `git.secretName` | Kubernetes secret with the SSH private key | None
-| `ssh.known_hosts`  | The contents of an SSH `known_hosts` file, if you need to supply host key(s) | None
+| `git.secretName` | Kubernetes secret with the SSH private key. Superceded by `helmOperator.git.secretName` if set. | `None`
+| `ssh.known_hosts`  | The contents of an SSH `known_hosts` file, if you need to supply host key(s) | `None`
 | `registry.pollInterval` | Period at which to check for updated images | `5m`
 | `registry.rps` | Maximum registry requests per second per host | `200`
 | `registry.burst` | Maximum number of warmer connections to remote and memcache | `125`
 | `registry.trace` |  Output trace of image registry requests to log | `false`
-| `registry.insecureHosts` | Use HTTP rather than HTTPS for these image registry domains | None
+| `registry.insecureHosts` | Use HTTP rather than HTTPS for these image registry domains | `None`
+| `registry.cacheExpiry` | Duration to keep cached image info (deprecated) | `None`
 | `memcached.verbose` | Enable request logging in memcached | `false`
 | `memcached.maxItemSize` | Maximum size for one item | `1m`
 | `memcached.maxMemory` | Maximum memory to use, in megabytes | `64`
-| `memcached.resources` | CPU/memory resource requests/limits for memcached | None
+| `memcached.resources` | CPU/memory resource requests/limits for memcached | `None`
 | `helmOperator.create` | If `true`, install the Helm operator | `false`
+| `helmOperator.createCRD` | Create the `v1beta1` and `v1alpha2` flux CRDs. Dependent on `helmOperator.create=true` | `true`
 | `helmOperator.repository` | Helm operator image repository | `quay.io/weaveworks/helm-operator`
 | `helmOperator.tag` | Helm operator image tag | `<VERSION>`
+| `helmOperator.replicaCount` | Number of helm operator pods to deploy | `1`
 | `helmOperator.pullPolicy` | Helm operator image pull policy | `IfNotPresent`
 | `helmOperator.updateChartDeps` | Update dependencies for charts | `true`
 | `helmOperator.git.pollInterval` | Period at which to poll git repo for new commits | `git.pollInterval`
 | `helmOperator.git.timeout` | Duration after which git operations time out | `git.timeout`
+| `helmOperator.git.secretName` | The name of the kubernetes secret with the SSH private key, supercedes `git.secretName` | `None`
 | `helmOperator.chartsSyncInterval` | Interval at which to check for changed charts | `3m`
 | `helmOperator.chartsSyncTimeout` | Timeout when checking for changed charts | `1m`
```

Comment:
@squaremo 
Is this now a defunct option ?

Can't find where it's actively used.

Reply:
It is used if you refer helm charts from git

Reply:
I cannot find `chartsSyncTimeout` in `values.yaml` [here](https://github.com/weaveworks/flux/blob/master/chart/flux/values.yaml)

Should it be added?

I also can't find any reference to it in the code.

Reply:
Hmm you highlighted the `helmOperator.git.secretName` not  `chartsSyncTimeout`

Reply:
The `chartsSyncTimeout` is gone, please remove it from the table.

Reply:
oops... ya, i meant `chartsSyncTimeout`